src/withdrawals/main.eas

```diff
@@ -361,15 +361,15 @@ update_excess:
 
   ;; Check if excess needs to be reset to 0 for first iteration after
   ;; activation.
-  dup1                  ;; [excess, excess, count, count]
-  push EXCESS_INHIBITOR ;; [inhibitor, excess, excess, count, count]
-  eq                    ;; [inhibitor == excess, excess, count, count]
-  iszero                ;; [inhibitor != excess, excess, count, count]
-  jumpi @skip_reset     ;; [excess, count, count]
+  dup1                  ;; [excess, excess, count]
+  push EXCESS_INHIBITOR ;; [inhibitor, excess, excess, count]
+  eq                    ;; [inhibitor == excess, excess, count]
+  iszero                ;; [inhibitor != excess, excess, count]
+  jumpi @skip_reset     ;; [excess, count]
 
-  ;; Drop the excess from stack and use 0.
-  pop                   ;; [count, count]
-  push 0                ;; [reset_excess, count]
+  ;; Drop the count from stack and use 0.
+  pop                   ;; [count]
+  push 0                ;; [reset_excess]
```

Comment:
This change looks incorrect. The POP instruction does drop `excess`, and then a zero is pushed to set it to zero.
So the comments should be like this:

```
  ;; Drop excess from the stack and replace it with zero.
  pop                   ;; [count]
  push 0                ;; [excess, count]
```